src/styles/moto_styles.scss

```diff
@@ -0,0 +1,79 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
```

Comment:
You have already set these parameters for all pages, no need to duplicate the code.